docs/user-guide/downward-api/volume/index.md

```diff
@@ -2,7 +2,7 @@
 ---
 
 Following this example, you will create a pod with a downward API volume.
-A downward API volume is a k8s volume plugin with the ability to save some pod information in a plain text file. The pod information can be  for example some [metadata](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api-conventions.md#metadata).
+A downward API volume is a k8s volume plugin with the ability to save some pod information in a plain text file. The pod information can be for example some [metadata](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api-conventions.md#metadata) or a container's [resources](http://kubernetes.io/docs/user-guide/compute-resources/).
```

Comment:
@johndmulhausen Can we do it? Or Am I understanding it wrong as this link is not local i.e. docs/user-guide/compute-resources/ is not local to kubernetes.github.io repo?